public/js/actions/subscriptions.js

```diff
@@ -98,11 +100,20 @@ export function _createSubscription({dispatch, productId,
     console.log('Successfully subscribed + completed payment');
     dispatch(transactionActions.complete({userEmail: email}));
   }).fail($xhr => {
-    if (data.pay_method_nonce) {
-      dispatch({
-        type: actionTypes.CREDIT_CARD_SUBMISSION_ERRORS,
-        apiErrorResult: $xhr.responseJSON,
-      });
+    console.log($xhr);
```

Comment:
Yeah that might be nice. Even if it's just status and responseJSON / responseText etc.